deploy/operator.yaml

```diff
@@ -2,15 +2,29 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: percona-server-mongodb-operator
+  labels:
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/instance: percona-server-mongodb-operator
+    app.kubernetes.io/name: percona-server-mongodb-operator
+    app.kubernetes.io/part-of: percona-server-mongodb-operator
+    app.kubernetes.io/version: "v1.18.0"
```

Comment:
@cshiv we can remove the versions from all the resources since it will be a manual process to update them and we can avoid this since we don't really need it.

Reply:
We can keep all the other labels and we are ready!

Reply:
Sounds good, lets do it

Reply:
@cshiv please remove  app.kubernetes.io/version label and we will merge it.

Reply:
@hors Removed the version